spec/views/admin/users/index.html.erb_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "admin/users/index", type: :view do
+  it "renders users index page" do
+    user = create(:user)
+    assign(:users, [user])
+
+    render
+
+    expect(rendered).to have_content("Admin Area")
+    expect(rendered).to have_content("Users")
+
+    expect(rendered).to have_link("Create new user", href: new_admin_user_path.to_s)
+
+    expect(rendered).to have_css("tr th", text: "Email")
+    expect(rendered).to have_css("tr td", text: user.email.to_s)
+
+    expect(rendered).to have_css("tr th", text: "Name")
+    expect(rendered).to have_css("tr td", text: user.name.to_s)
+
+    expect(rendered).to have_css("tr th", text: "Created at")
+    expect(rendered).to have_css("tr td", text: user.created_at.strftime("%d %b %H:%M").to_s)
+    expect(rendered).to have_css("[data-react-component=Link]", count: 2)
```

Comment:
Not sure what is the best way to test react_components in view specs

Reply:
I think this is the best we can do without going down the rabbit hole of checking all props etc.
However: We could have just used the BEM-classes for these links instead of `react_component`, and in that case it would have been easier to test.

Reply:
I've replaced the react components for these links with plain links using the relevant "Link" classes